01-HTML-CSS-Basics/second.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Second HTML5 Page</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <style>
+            table, th, td {
+                text-align: center;
+                border: 1px solid black;
+                border-collapse: collapse;
+            }
+            th, td {
+                padding: 15px;
+            }
+            
+        </style>
+    </head>
+    <body>
+        <div>
+            <table>
+                <tr>
+                    <th>Expenses</th>
```

Comment:
A better practice would be having the <th> inside a <thead> tag. And the rest inside a <tbody> tag.